### PR TITLE
add product repository

### PR DESCRIPTION
**Pull request by duongricky**

### CODE REVIEW

**Comment by duongricky** [comment]
ready

**Comment by duongricky** [comment]
ready

File: app/Http/Controllers/Client/OrderController.php [hunk 1]
```diff
@@ -24,15 +23,17 @@ class OrderController extends Controller
     protected $orderRepo;
     protected $orderInforRepo;
     protected $orderDetailRepo;
+    protected $productRepo;
 
     public function __construct(OrderRepositoryInterface $orderRepo, OrderInforRepositoryInterface $orderInforRepo,
-        OrderDetailRepositoryInterface $orderDetailRepo
+        OrderDetailRepositoryInterface $orderDetailRepo, ProductRepositoryInterface $productRepo
     )
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Client/OrderController.php]:
```suggestion
    ) {
```

**Comment by duongricky** [comment]
ready

**Pull request merged**